rust/src/treefile.rs

```diff
@@ -516,6 +524,33 @@ for x in *; do mv ${{x}} %{{buildroot}}%{{_prefix}}/lib/ostree-jigdo/%{{name}};
     }
 }
 
+fn hash_file(hasher: &mut glib::Checksum, mut f: &fs::File) -> Fallible<()> {
+    let mut reader = io::BufReader::new(f.try_clone()?);
+    let mut buf = Vec::new();
+    reader.read_to_end(&mut buf)?;
```

Comment:
Doesn't matter a lot here but probably worth a comment like
`// If we were to use this for large files we should read into a e.g. 128k buffer`

Reply:
For fun, I switched to chunked reads below! :arrow_down: